src/VisualStudio/Core/Def/Implementation/TaskList/ExternalErrorDiagnosticUpdateSource.cs

```diff
@@ -320,6 +336,9 @@ private async Task SyncBuildErrorsAndReportOnBuildCompletedAsync(DiagnosticAnaly
 
                 ReportBuildErrors(group.Key, solution, group.ToImmutableArray());
             }
+
+            // Report pending live errors
+            await diagnosticService.SynchronizeWithBuildAsync(_workspace, pendingLiveErrorsToSync, onBuildCompleted: true, inProgressState.CancellationToken).ConfigureAwait(false);
```

Comment:
Moved this below as deduping and updating live errors is more expensive than reporting build-only errors.